.vscode/settings.json

```diff
@@ -41,23 +39,7 @@
   // Terminal
   //////////////////////////////////////////////////////////////////////////////
 
-  "terminal.integrated.profiles.linux": {
-    "fish": {
-      "path": "fish"
-    }
-  },
-
-  "terminal.integrated.automationProfile.linux": {
-    "path": "fish",
-    "icon": "terminal"
-  },
-
-  "terminal.integrated.env.linux": {
-    // TODO: Determine whether defining env-vars here would be preferable.
-  },
```

Comment:
All these fields were complaining about going to either be deprecated soon or currently were deprecated&hellip;

Reply:
FWIW, I've never used this and never had trouble with Fish and VSCode built-in's terminal.